roles/azure_controllers/tasks/main.yml

```diff
@@ -69,24 +106,25 @@
     loop_var: instance_item
   when: vsmart_instances is defined and (instance_item.hostname not in instances_info or not instances_info[instance_item.hostname])
 
-- name: Deploy vManage - Create Virtual Machines
+- name: "Deploy vManage - Create Virtual Machines"
   ansible.builtin.include_tasks: azure_vmanage_vm.yml
   vars:
     hostname: "{{ instance_item.hostname }}"
     system_ip: "{{ instance_item.system_ip }}"
     site_id: "{{ instance_item.site_id }}"
+    persona: "{{ instance_item.persona | default('COMPUTE_AND_DATA') }}"
```

Comment:
Instead of providing default value in tasks file, maybe it would be better to have it in defaults/main.yaml or any other defaults file?

Reply:
good idea - changed in latest update